generator/src/main/java/org/corfudb/generator/LongevityApp.java

```diff
@@ -0,0 +1,131 @@
+package org.corfudb.generator;
+
+import ch.qos.logback.classic.Logger;
+import lombok.extern.slf4j.Slf4j;
+import org.corfudb.generator.operations.CheckpointOperation;
+import org.corfudb.generator.operations.Operation;
+import org.corfudb.runtime.CorfuRuntime;
+import org.slf4j.LoggerFactory;
+
+import java.util.concurrent.ArrayBlockingQueue;
+import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Created by rmichoud on 7/25/17.
+ */
+
+/**
+ * This Longevity app will stress test corfu using the Load
+ * generator during a given duration.
+ *
+ * Current implementation spawns a task producer and a task
+ * consumer. Both of them are a single threaded. In combination
+ * with the blocking queue, they limit the amount of task created
+ * during execution. Apps thread pool will take care of executing
+ * the operations.
+ *
+ */
+@Slf4j
+public class LongevityApp {
+    private long durationMs;
+    private String configurationString;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "configurationString" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)